extensions/2.0/Khronos/KHR_image_formation/README.md

```diff
@@ -0,0 +1,197 @@
+<!--
+Copyright 2018-2020 The Khronos Group Inc.
+SPDX-License-Identifier: LicenseRef-KhronosSpecCopyright
+-->
+
+# KHR\_image\_formation
+
+## Contributors
+
+* Romain Guy, Google [@romainguy](https://twitter.com/romainguy)
+* Emmett Lalish, Google [@emmettlalish](https://twitter.com/emmettlalish)
+
+Copyright 2018-2020 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+The scene referred values computed during the lighting pass using the glTF 2.0 spec are luminance values, expressed in cd.m-2 or nits. It follows that light incident to our virtual optical system can cover a large range of values when physical light units (KHR_lights_punctual) are used, from 10-5 cd.m-2 for starlight to 109cd.m-2 for the sun. However the common methods we have of storing and displaying images have limited dynamic range. Of course this problem does not just apply to rendering; the human eye also has limited dynamic range, and of course photographs printed on paper (or displayed on a screen) cannot possibly reproduce the range  of intensity that was incident on the lens. 
+
+This extension is intended to allow the producer of a glTF to define all the parameters necessary for a consumer (renderer) to generate a consistent output image of the scene, given consistent lighting and view. Consistent does not imply pixel-identical, as the rendering is still subject to many arbitrary approximations, but simply that the output would be identical if the physics were respected exactly. What this extension defines is a function that maps [0, infinity) input light RGB to the [0, 1] output encoding color gamut (most commonly sRGB). In practice, this function has a much larger effect on the appearance of the output than the differences in renderer approximations.
```

Comment:
I guess this only holds true when models are not mixed, ie adding a model from another glTF that also uses this extension but with a different configuration - would not result in consistent output.
Is this a correct assumption?

Reply:
Could you please elaborate on the usecase for this extension, who are it's expected users?

Reply:
Am I also correct assuming that consistent output will not be achieved when light conditions change, ie intensity of light changes or lights are removed or added?

Reply:
I think this is all addressed in the next paragraph and our discussion of it.

Reply:
Could you please elaborate on the usecase for this extension?

Reply:
I'm going to resolve this one and answer below, to avoid having two threads on the same topic.